lib/ecto/migrator.ex

```diff
@@ -687,11 +687,12 @@ defmodule Ecto.Migrator do
           * There is a deadlock while migrating (such as using concurrent
             indexes with a migration_lock)
 
-        To fix the first issue, run "mix ecto.create".
+        To fix the first issue, run "mix ecto.create" for the desired MIX_ENV.
 
         To address the second, you can run "mix ecto.drop" followed by
-        "mix ecto.create". Alternatively you may configure Ecto to use
-        another table and/or repository for managing migrations:
+        "mix ecto.create", both for the desired MIX_ENV. Alternatively you may
+        configure Ecto to use another table and/or repository for managing
+        migrations:
```

Comment:
FWIW, I considered interpolating the actual `MIX_ENV=test ` prefix for these commands via a module attribute, but it'd be hard to write tests for conditionality, due to the conditionality being triggered off of `Mix.env()` at compile time.

If there _is_ an appetite for interpolating the prefix, I'd be happy to open a follow-up.